modular_skyrat/modules/colony_fabricator/code/tools/tools.dm

```diff
@@ -0,0 +1,135 @@
+	custom_materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 1.75,
+		/datum/material/silver = HALF_SHEET_MATERIAL_AMOUNT * 1.5,
+		/datum/material/titanium = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	force = 10
+	throwforce = 8
+	throw_speed = 2
+	throw_range = 3
+	attack_verb_continuous = list("drills", "screws", "jabs", "whacks")
+	attack_verb_simple = list("drill", "screw", "jab", "whack")
+	hitsound = 'sound/items/drill_hit.ogg'
+	usesound = 'sound/items/drill_use.ogg'
+	w_class = WEIGHT_CLASS_SMALL
+	toolspeed = 1
+	random_color = FALSE
+	greyscale_config = null
+	greyscale_config_belt = null
+	greyscale_config_inhand_left = null
+	greyscale_config_inhand_right = null
+
+/obj/item/screwdriver/omni_drill/get_all_tool_behaviours()
+	return list(TOOL_WIRECUTTER, TOOL_SCREWDRIVER, TOOL_WRENCH)
+
+/obj/item/screwdriver/omni_drill/examine(mob/user)
+	. = ..()
+	. += span_notice("Use <b>in hand</b> to switch configuration.")
+	. += span_notice("It functions as a <b>[tool_behaviour]</b> tool.")
+	. += span_danger("<i>-100% random critical hit chance.</i>")
+
+/obj/item/screwdriver/omni_drill/update_icon_state()
+	. = ..()
+	switch(tool_behaviour)
+		if(TOOL_SCREWDRIVER)
+			icon_state = initial(icon_state)
+		if(TOOL_WRENCH)
+			icon_state = "[initial(icon_state)]_bolt"
+		if(TOOL_WIRECUTTER)
+			icon_state = "[initial(icon_state)]_cut"
+
+/obj/item/screwdriver/omni_drill/attack_self(mob/user, modifiers)
+	. = ..()
+	if(!user)
+		return
+	var/list/tool_list = list(
+		"Screwdriver" = image(icon = icon, icon_state = "drill"),
+		"Wrench" = image(icon = icon, icon_state = "drill_bolt"),
+		"Wirecutters" = image(icon = icon, icon_state = "drill_cut"),
+		)
+	var/tool_result = show_radial_menu(user, src, tool_list, custom_check = CALLBACK(src, PROC_REF(check_menu), user), require_near = TRUE, tooltips = TRUE)
+	if(!check_menu(user) || !tool_result)
+		return
+	switch(tool_result)
+		if("Wrench")
+			tool_behaviour = TOOL_WRENCH
+			sharpness = NONE
+		if("Wirecutters")
+			tool_behaviour = TOOL_WIRECUTTER
+			sharpness = NONE
+		if("Screwdriver")
+			tool_behaviour = TOOL_SCREWDRIVER
+			sharpness = SHARP_POINTY
+	playsound(src, 'sound/items/change_drill.ogg', 50, vary = TRUE)
+	update_appearance(UPDATE_ICON)
+
+/obj/item/screwdriver/omni_drill/proc/check_menu(mob/user)
+	if(!istype(user))
+		return FALSE
+	if(user.incapacitated() || !user.Adjacent(src))
+		return FALSE
+	return TRUE
+
+// Just a completely normal crowbar except its normal sized and can force doors like jaws of life can
+
+/obj/item/crowbar/large/doorforcer
+	name = "prybar"
+	desc = "A large, sturdy crowbar, painted orange. This one just happens to be tough enough to \
+		survive <b>forcing doors open</b>."
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/tools.dmi'
+	icon_state = "prybar"
+	toolspeed = 1
+	force_opens = TRUE
+	custom_materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 1.75,
+		/datum/material/titanium = HALF_SHEET_MATERIAL_AMOUNT,
+	)
```

Comment:
Okay, but why would we want to make this available to anyone except engineering?

I can get behind the omnidrill, but this just feels like "oh I wanted the omni drill but now I need something else for the crowbar otherwise it's boring"

Reply:
The door forcer crowbar is something I planned before the omni drill actually, and its another "I don't think it'll be a problem" and as junglerat said in the contributor channel, if you wanted to get through a door there are better ways like just getting an rcd and deleting the door, or hacking it with the free greybull you can get everywhere.

Reply:
As discussed, this wouldn't be freely accessible by all of the crew from the console, they'd have to go through to the console inside of cargo in order to order it.